realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -282,6 +284,46 @@ public void refresh() {
     }
 
     /**
+     * Blocks the current thread until new changes to the Realm are available or {@link #stopWaitForChange}
```

Comment:
Replace second sentence with something like:

"Once this method is called all future calls to waitForChange will return false immediately."